tests/test_request.py

```diff
@@ -37,19 +34,10 @@ def test__init__(self):
 
 
 class TestRequest:
-    def setup_method(self):
+    @pytest.fixture(autouse=True)
```

Comment:
You shouldn't mark `setup_method` as a fixture - that is a wrong usage of this functionality. You should prepare a normal fixture above which will be used in the tests:
```python
@pytest.fixture
def sample_request(user: User) -> Request:
    return Request(
        ...
        user=user,
    )
```

Reply:
Or, if you want to keep the `setup_method` as it is right now, you should prepare a factory that can be used within this method. But it will be rather a normal function - it won't break anything in the `pytest` fixtures 😉